providers/fips/fipsprov.c

```diff
@@ -198,7 +200,7 @@ static int fips_get_params_from_core(FIPS_GLOBAL *fgbl)
     * OSSL_PROV_FIPS_PARAM_SECURITY_CHECKS and
     * OSSL_PROV_FIPS_PARAM_TLS1_PRF_EMS_CHECK are not self test parameters.
     */
-    OSSL_PARAM core_params[26], *p = core_params;
+    OSSL_PARAM core_params[28], *p = core_params;
```

Comment:
Why `[28]`? Only one additional parameter was added. Shouldn't this be `[27]`?

Reply:
I was attempting to avoid a conflict with one of my other in flight PRs.

Yes, it should be 27 but since we get a conflict ever merge, it will end up correct.
I've been tempted to submit a PR converting this to a stack of params.  More than once.

Reply:
After merging #24120, it now needs to be 28 :grin: